src/core/CoolBoard.cpp

```diff
@@ -458,6 +457,28 @@ void CoolBoard::spiffsProblem() {
   }
 }
 
+void CoolBoard::lowBattery() {
+  WiFi.mode(WIFI_OFF);
+  SPIFFS.end();
+  for (uint8_t i = 0; i < 8; i++) {
+    this->coolBoardLed.blink(RED, 0.2);
+    if (i <= 8) {
+      ESP.deepSleep(LOW_POWER_SLEEP);
+    }
+  }
+}
+
+bool CoolBoard::powerCheck() {
```

Comment:
this shouldn't return a boolean. this should be `void powerCheck()` and just sleep if battery too low. Otherwise you will never see the result of the boolean check as you will start from the beginning